tests/GatewayTest.php

```diff
@@ -10,6 +10,11 @@
 class GatewayTest extends TestCase {
     const HOST = 'api.staging-rumbleship.com';
 
+    // required for testing overriding Api->login()
+    const TEST_ID_TOKEN= 'api123key';
```

Comment:
Do these need to be at this level?

Reply:
Nope. I don't expect other `GatewayTest` methods to require them; moved inline to test methods.